iothub/device/src/Transport/AmqpIoT/AmqpIoTErrorAdapter.cs

```diff
@@ -25,7 +25,6 @@ internal static class AmqpIoTErrorAdapter
         public static readonly AmqpSymbol IotHubNotFoundError = AmqpIoTConstants.Vendor + ":iot-hub-not-found-error";
         public static readonly AmqpSymbol ArgumentError = AmqpIoTConstants.Vendor + ":argument-error";
         public static readonly AmqpSymbol ArgumentOutOfRangeError = AmqpIoTConstants.Vendor + ":argument-out-of-range";
-        public static readonly AmqpSymbol DeviceAlreadyExists = AmqpIoTConstants.Vendor + ":device-already-exists";
```

Comment:
You have a confirmation from partners if this is fine to remove?

Reply:
We have confirmation from the service team that they never issue an error of `com.microsoft:device-already-exists` over Amqp, on the device client API: https://github.com/Azure/azure-iot-sdk-csharp/pull/1504#discussion_r465380144

Reply:
Also, any error code not explicitly mapped by our SDK would be translated to a non-retryable `IotHubException`.